modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/index.js

```diff
@@ -31,20 +31,65 @@ const SetupDXPCloudModal = ({
 	observer,
 	onClose,
 	project,
+	setVisibleModal,
 	subscriptionGroupId,
 }) => {
+	const [formAlreadySubmitted, setFormAlreadySubmitted] = useState(false);
+
 	return (
 		<ClayModal center observer={observer}>
-			<SetupDXPCloud
-				handlePage={onClose}
-				leftButton="Cancel"
-				project={project}
-				subscriptionGroupId={subscriptionGroupId}
-			/>
+			{formAlreadySubmitted ? (
+				<AlreadySubmittedFormModal setVisibleModal={setVisibleModal} />
+			) : (
+				<SetupDXPCloud
+					handlePage={onClose}
+					leftButton="Cancel"
+					project={project}
+					setFormAlreadySubmitted={setFormAlreadySubmitted}
+					subscriptionGroupId={subscriptionGroupId}
+				/>
+			)}
 		</ClayModal>
 	);
 };
 
+const AlreadySubmittedFormModal = ({setVisibleModal}) => {
+	return (
+		<div className="pt-4 px-4">
+			<div className="flex-row mb-2">
+				<header className="mb-5">
+					<h2 className="mb-1 text-neutral-10">Set up DXP Cloud</h2>
+
+					<p className="text-neutral-7 text-paragraph-sm">
+						We&#39;ll need a few details to finish building your DXP
+						environment(s).
+					</p>
+				</header>
+
+				<h5 className="mb-1 mt-1 text-neutral-10">
```

Comment:
I think you could use only my-1 instead of margin bottom and margin top

Reply:
Done! Thank you!